### ISSUE TITLE
Duplicate bar at end of chart

### ISSUE THREAD

**Issue by phong-phuong** [HusariaSolutions/solid-crypto-chart#1]
To reproduce: wait for chart to load, wait for 1st tick to update. A new bar will appended to the end instead of updating current bar.

![image](https://user-images.githubusercontent.com/66398960/178668755-355085c7-b125-4796-a043-5238f4e51a3b.png)

**Comment by phong-phuong** [comment]
To fix this, use small t, not big T in adaptor.ts. Big T is current timestamp., small t is k line open time.

```
export function klinesWebSocketDataAdaptor(data: KlinesWebSocketData): CandlestickData & { value: number } {
  const {
    k: {t, o, c, h, l, v, V }, //  <--- changed from T to t
  } = data

  return {
    open: parseFloat(o),
    high: parseFloat(h),
    low: parseFloat(l),
    close: parseFloat(c),
    time: (t/ 1000) as Time, // <--- changed from T to t
    value: (parseFloat(v) + parseFloat(V)) / 2,
    color: o < c ? '#005a40' : '#82112b',
  }
}
```

Also update types.ts:

```
  export type KlinesWebSocketData = {
    k: {t: number; o: string; c: string; h: string; l: string; v: string; V: string } // <--- changed from T to t
  }
```

**Comment by DevHusariaSolutions** [comment]
Nice, thanks for catching that.
Your PR has been merged, changes available in 0.0.3 version.
Pls message us via contact@husariasolutions.com if interested in contribution of this or crypto related packages.
